framework/src/media/utils/MediaUtils.cpp

```diff
@@ -420,7 +428,8 @@ bool header_parsing(unsigned char *buffer, unsigned int bufferSize, audio_type_t
 		isHeader = false;
 		headPoint = 0;
 		while (headPoint < bufferSize) {
-			if (buffer[headPoint] == 0xFF) {
+			/* 11 bits for MP3 Sync Word(the beginning of the frame) */
+			if (buffer[headPoint] == 0xFF && ((buffer[headPoint + 1] & 0xE0) == 0xE0)) {
```

Comment:
add one bracket set for first condition.

Reply:
i added bracket